src/routes/onboarding/set-exercise-goals/set-exercise-goals-form.tsx

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+import { Form } from "../../../components"
+import { SetExerciseGoals } from "./set-exercise-goals-content";
+
+export const SetExerciseGoalsForm: React.FC = () => {
+  return (
+    <div className="flex flex-col bg-lightBlue max-w-screen-md px-[10px] relative m-auto">
+      <h2 className="font-text text-section-header font-medium leader-[38.4px] text-center max-w-[358px] mx-auto">Set your exercise goals</h2>
```

Comment:
Misspelling here. 
Line height class is leading, not leader, so it needs to be leading-[38.4px] (I would probably round it down to 38px too).

In addition, you are missing text-black to make the text #121212 like the figma requires.